Add instructions on how to use docs-xvfb (#138)
napari/docs: docs/developers/documentation/index.md

on your terminal.
````

````{tip}
If you have [xvfb](https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml)
installed on your system, you can also run a "headless GUI" build by using the
`docs-xvfb` command:

```shell
$ make docs-xvfb
```

> This will prevent all but the first napari window to be shown during the docs

Reviewer: ```suggestion
This will prevent all but the first napari window from being shown during the docs
```
